main.py

```diff
@@ -58,39 +68,54 @@ def draw(state, a, b, colour):
         pygame.draw.rect(Screen, colour, (x, y, s, s))
 
 
-# noinspection PyShadowingNames
-def check_user_input(board, paused):
+def check_user_input(board, paused, fps, fps_limit):
     """Checks for user input and acts accordingly"""
     one_turn = False
     for event in pygame.event.get():
         x, y = pygame.mouse.get_pos()
         a = maths.floor(x / config.Size) + config.Cushion
         b = maths.floor(y / config.Size) + config.Cushion
```

Comment:
https://stackoverflow.com/questions/19507808/python3-integer-division does this work here?

Reply:
done